lib/gitlab/client/runners.rb

```diff
@@ -0,0 +1,124 @@
+      get("/runners/#{id}")
+    end
+
+    # Update details of a runner.
+    # Full runner params documentation: https://github.com/gitlabhq/gitlabhq/blob/master/doc/api/runners.md
+    #
+    # @example
+    #   Gitlab.update_runner(42, { description: 'Awesome runner' })
+    #   Gitlab.update_runner(42, { active: false })
+    #   Gitlab.update_runner(42, { tag_list: [ 'awesome', 'runner' ] })
+    #
+    # @param  [Integer, String] id The ID of a runner
+    # @param  [Hash] options A customizable set of options.
+    # @option options [String] :active The state of a runner; can be set to true or false.
+    # @option options [String] :tag_list The list of tags for a runner; put array of tags, that should be finally assigned to a runner
+    # @return <Gitlab::ObjectifiedHash>
+    def update_runner(id, options={})
+      put("/runners/#{id}", query: options)
+    end
+
+    # Remove a runner.
+    # Full runner params documentation: https://github.com/gitlabhq/gitlabhq/blob/master/doc/api/runners.md
+    #
+    # @example
+    #   Gitlab.delete_runner(42)
+    #
+    # @param  [Integer, String] id The ID of a runner
+    # @return <Gitlab::ObjectifiedHash>
+    def delete_runner(id)
+      delete("/runners/#{id}")
+    end
+
+    # List all runners (specific and shared) available in the project. Shared runners are listed if at least one shared runner is defined and shared runners usage is enabled in the project's settings.
+    # Full runner params documentation: https://github.com/gitlabhq/gitlabhq/blob/master/doc/api/runners.md
+    #
+    # @example
+    #   Gitlab.project_runners(42)
+    #
+    # @param  [Integer, String] id The ID of a project.
+    # @return [Array<Gitlab::ObjectifiedHash>]
+    def project_runners(project_id)
+      get("/projects/#{project_id}/runners")
+    end
+
+    # Enable an available specific runner in the project.
+    # Full runner params documentation: https://github.com/gitlabhq/gitlabhq/blob/master/doc/api/runners.md
+    #
+    # @example
+    #   Gitlab.project_enable_runner(2, 42)
+    #
+    # @param  [Integer, String] id The ID of a project.
+    # @param  [Integer, String] id The ID of a runner.
+    # @return <Gitlab::ObjectifiedHash>
+    def project_enable_runner(project_id, id)
+      body = { runner_id: id }
+      post("/projects/#{project_id}/runners", body: body)
+    end
+
+    # Disable a specific runner from the project. It works only if the project isn't the only project associated with the specified runner.
+    # Full runner params documentation: https://github.com/gitlabhq/gitlabhq/blob/master/doc/api/runners.md
+    #
+    # @example
+    #   Gitlab.project_disable_runner(2, 42)
+    #
+    # @param  [Integer, String] id The ID of a project.
+    # @param  [Integer, String] id The ID of a runner.
+    # @return <Gitlab::ObjectifiedHash>
+    def project_disable_runner(project_id, id)
+      body = { runner_id: id }
+      delete("/projects/#{project_id}/runners/#{id}", body: body)
```

Comment:
Request body is not needed to disable a runner according to this doc: https://github.com/gitlabhq/gitlabhq/blob/master/doc/api/runners.md.  Have you observed something different?